submissions/sprint_a/__init__.py

```diff
@@ -0,0 +1,19 @@
+
+def configure_app():
+    import os
+    from dotenv import load_dotenv
+    from flask import Flask
+
+    PROJECT_ROOT = os.path.dirname('.')
+    
+    dotenv_path = os.path.abspath(
+        os.path.join(PROJECT_ROOT, '.', '.env')
```

Comment:
It's not necessary to do this. The period in this case should be ignored when traversing the absolute path formed from joining PROJECT_ROOT with the the file name.